eng/references.targets

```diff
@@ -1,67 +1,58 @@
 <Project>
   <PropertyGroup>
-    <ContractDependencyPaths>$(RefPath)</ContractDependencyPaths>
-    <FrameworkPathOverride>$(RefPath)</FrameworkPathOverride>
-    <AssemblySearchPaths>$(AssemblySearchPaths);$(RefPath);{RawFileName}</AssemblySearchPaths>
+    <IncludeDefaultReferences Condition="'$(IncludeDefaultReferences)' == ''">true</IncludeDefaultReferences>
```

Comment:
Do we want to include default references for `NETCoreAppCurrent`?

Reply:
I see we need it in order to use the ref and runtime pack.

Reply:
`IncludeDefaultReferences` is a historical property that only defines if the netstandard reference should be added. There are only very few places that use that property: https://github.com/dotnet/runtime/search?q=IncludeDefaultReferences&unscoped_q=IncludeDefaultReferences. The shims wouldn't event need that property as `IsSourceProject` isn't true for them.

I might clean this up as part of this PR if I have time.

Reply:
Looks like you added that property: https://github.com/dotnet/runtime/commit/6ec72ff7ea729936438de20cb95ecba8e710e9d9#diff-d3ed5cb795a36ae602695c7d88edbe72R7

Reply:
Ok deleted that code as the property isn't needed anymore. Thanks.